src/_typography.scss

```diff
@@ -38,6 +38,33 @@
   }
 }
 
+@mixin link {
+  color: $c-primary;
+  font-family: $ff-sans-serif;
+  font-weight: normal;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: $lh-l;
+  letter-spacing: normal;
+  text-align: left;
+
+  &:hover,
+  &:active {
+    color: $c-primary-dark;
+    text-decoration: underline;
+  }
+}
+
+@mixin link-negative {
```

Comment:
what about a simple `mixin` with types (default & negative) as button? https://github.com/SUI-Components/sui-theme/blob/master/src/components/_button.scss

Reply:
I like to have a unique mixin because, with a flag (for example `isNegative` or `negative`), we could set the CSS property `color` without overwriting styles.

Reply:
You could approach it like the following example:
```sass
@mixin link($negative: false) {
  color: if($negative, $c-white, $c-primary);
  font-family: $ff-sans-serif;
  font-weight: normal;
  font-style: normal;
  font-stretch: normal;
  line-height: $lh-l;
  letter-spacing: normal;
  text-align: left;

  &:hover,
  &:active {
    color: if($negative, $c-white, $c-primary-dark);
    text-decoration: underline;
  }
}
```